### PR TITLE
Add blog post drafts

### PR DESCRIPTION
**Pull request by adelpopelkova**
Issue https://github.com/encukou/freezeyt/issues/1

### CODE REVIEW

File: freezeyt_blog/articles/meeting08.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+
+## Nastavování defaultních portů
+Začali jsme opravováním nového testu, který přidával možnost použít url bez uvedení portu. Přidali jsme do funkcí `freeze` a `url_to_filename` přidání defaultního portu. A následně přišel na řadu „třetí“ krok TDD, a to refactoring. Momentálně se nám při nezadání portu daný port vyhodnocoval dvakrát. Rozhodli jsme se, že nejlepším řešením bude přidat funkci, která URL adresu zpracuje. Parse result dědí z ntice, takže je nemění, nemůžeme tedy přidat/změnit port. Dědí z třídy named tuple, metoda [*_replace*](https://docs.python.org/3.8/library/collections.html#collections.somenamedtuple._replace) z modulu collections (ze standardní knihovny) by nám mohla pomoci s měněním výchozích portů. `_replace` pracuje na pojmenované ntici, jenže port je atribut třídy, takže není v ntici. Bylo by dobré napsat [unit testy](To-do - najít odkaz) na tuto funkci, takže úkol do příště.
```
Comment (encukou) [inline freezeyt_blog/articles/meeting08.md:24]:
```suggestion
Začali jsme opravováním nového testu, který přidával možnost použít url bez uvedení portu. Přidali jsme do funkcí `freeze` a `url_to_filename` přidání defaultního portu. A následně přišel na řadu „třetí“ krok TDD, a to refactoring. Momentálně se nám při nezadání portu daný port vyhodnocoval dvakrát. Rozhodli jsme se, že nejlepším řešením bude přidat funkci, která URL adresu zpracuje. Parse result dědí z ntice, takže je nemění, nemůžeme tedy přidat/změnit port. Dědí z třídy named tuple, metoda [*_replace*](https://docs.python.org/3.8/library/collections.html#collections.somenamedtuple._replace) z modulu collections (ze standardní knihovny) by nám mohla pomoci s měněním výchozích portů. `_replace` pracuje na pojmenované ntici, jenže port je atribut třídy, takže není v ntici. Bylo by dobré napsat *unit testy* (To-do - najít odkaz) na tuto funkci, takže úkol do příště.
```

Comment (adelpopelkova) [reply freezeyt_blog/articles/meeting08.md]: Thank You, before I found this suggestion I added links to Wikipedia and they can be changed later.

**Comment by adelpopelkova** [comment]
These are all the blog post drafts I found. Most of them were already on Slack. All of them should have a notice about the article not being finished.

**Review by encukou — APPROVED** [review]
Thank you!

**Pull request merged**